examples/meta-counter/src/contract.rs

```diff
@@ -87,6 +87,10 @@ impl Contract for MetaCounterContract {
             }
             Message::Increment(value) => {
                 let counter_id = self.counter_id();
+                // Make a service query: The result will be logged in the executed block.
+                let _ = self
+                    .runtime
+                    .query_service(counter_id, "query { value }".into());
```

Comment:
Could we have a test of the `fetch_json`?

Do only if there is a simple way (I can't think of one right now).

Reply:
I also can't think of a straightforward way to test this that would be much easier than moving some of the `linera-ethereum` logic into `linera-sdk` and writing a contract that uses it.

Reply:
Yes, my job now.